package-lock.json

```diff
@@ -8188,7 +8188,6 @@
           "version": "2.0.1",
```

Comment:
unless something installed new, package-lock.json should change. Please confirm.

Reply:
not sure how that changes but i have reverted the changes

Reply:
package-lock.json is only supposed to change if you add any additional libs(normal case). So if you haven't done it, then practically there shouldn’t be any change in the package-lock.json.

Reply:
agreed